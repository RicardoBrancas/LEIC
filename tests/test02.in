A ZSC 1
A AFR 5
A CCW 12
A VEN 7
F AFR ZSC
A ZWC 10
G ZSC AFR
R ZWC ZSC
F AFR CCW
F AFR VEN
G ZSC VEN
P
G CCW ZWC
V
G AFR ZWC
G AFR ZWC
G VEN ZWC
F ZSC VEN
C VEN
L 0
P
N CCW AFR
R ZSC CCW
R ZWC CCW
S ZSC ZWC
S AFR ZWC
R AFR ZWC
G VEN AFR
G VEN ZWC
G ZWC ZSC
Q
S VEN ZWC
I ZWC 7
F AFR CCW
G CCW AFR
O VEN
F VEN ZWC
F VEN ZWC
F VEN ZWC
G VEN ZWC
Q
P
V
L 0
N VEN ZWC
N ZWC VEN
S VEN ZWC
S VEN ZWC
S VEN ZWC
R VEN ZWC
L 0
F VEN ZWC
F VEN ZWC
F VEN ZWC
G VEN ZWC
S ZWC VEN
S ZWC VEN
R VEN ZWC
S ZWC VEN
X