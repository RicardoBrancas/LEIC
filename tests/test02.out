*Impossivel adicionar voo RT AFR ZSC
*Impossivel remover voo ZWC ZSC
*Impossivel adicionar voo ZSC VEN
Aeroporto com mais rotas AFR:2:3
Voo mais popular ZSC:AFR:1
*Impossivel adicionar voo AFR ZWC
*Impossivel adicionar voo AFR ZWC
*Impossivel adicionar voo RT ZSC VEN
ZSC:1:1:0
AFR:5:1:2
CCW:12:2:1
VEN:7:0:0
ZWC:10:0:1
Aeroporto com mais rotas AFR:1:2
Voos entre cidades CCW:AFR:1:1
*Impossivel remover voo ZSC CCW
*Impossivel remover voo ZWC CCW
*Impossivel remover voo RT ZSC ZWC
*Impossivel remover voo RT AFR ZWC
*Impossivel remover voo AFR ZWC
*Impossivel adicionar voo VEN AFR
*Impossivel adicionar voo VEN ZWC
*Impossivel adicionar voo ZWC ZSC
Aeroporto com mais ligacoes AFR:2
*Impossivel remover voo RT VEN ZWC
*Impossivel adicionar voo CCW AFR
Aeroporto com mais ligacoes AFR:2
Aeroporto com mais rotas ZWC:3:5
Voo mais popular VEN:ZWC:4
ZSC:1:1:0
AFR:5:2:3
CCW:12:3:2
VEN:7:4:3
ZWC:17:3:5
Voos entre cidades VEN:ZWC:4:3
Voos entre cidades ZWC:VEN:3:4
ZSC:1:1:0
AFR:5:2:3
CCW:12:3:2
VEN:7:0:0
ZWC:17:0:1
6:5
